lib/breezy_pdf_lite/intercept/html.rb

```diff
@@ -8,12 +8,15 @@ def call
 
       render_request = BreezyPDFLite::RenderRequest.new(body).submit
 
+      expires = 10.minutes.from_now.utc.strftime("%a, %e %b %Y %H:%M:%S GMT")
+
       [
         201,
         {
           "Content-Type" => "application/pdf",
           "Content-Length" => render_request.header["Content-Length"],
-          "Content-Disposition" => render_request.header["Content-Disposition"]
+          "Content-Disposition" => render_request.header["Content-Disposition"],
+          "Set-Cookie" => "breezy_pdf_downloaded=#{DateTime.now.to_i}; Expires=#{expires}"
```

Comment:
This cookie is not unique to the render request, therefore it would fail to be effective with > 1 requests.